org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/utils/MarkupContentFactory.java

```diff
@@ -57,4 +59,29 @@ public static MarkupContent createMarkupContent(String value, String preferredKi
 		}
 		return content;
 	}
+
+	/**
+	 * Create the markup content according the given markup kind and the capability
+	 * of the client.
+	 * 
+	 * @param values         the list of documentation values
+	 * @return the markup content according the given markup kind and the capability
+	 *         of the client.
+	 */
+	public static MarkupContent creatMarkupContent(List<String> values, IMarkupKindSupport request) {
+		if (values.size() ==1) {
+			if (request.canSupportMarkupKind(MarkupKind.MARKDOWN)) {
+				return new MarkupContent(MarkupKind.MARKDOWN, values.get(0));
+			}
+			return new MarkupContent(MarkupKind.PLAINTEXT, values.get(0));
+		}
+		String retValue = "";
+		for (String value : values) {
+			retValue = retValue + value + System.lineSeparator();

```

Comment:
I think we should separate several hover value (only in markdown?) with '___' like vscode does 
https://github.com/microsoft/vscode-languageserver-node/pull/417#issuecomment-447361795

We need to see the result in Eclipse IDE and vscode.

Reply:
OK, got it. I will update this PR with all above problems resolved.

Reply:
It seems '___' only works in markdown, so what about plaintext?

Reply:
Hi, I am wandering if we use '___' to separate hover values, we need to handle it on LSP client, right? otherwise it will looks very ugly.

Reply:
My idea was to generate line breaks (only if client can support markdown) https://www.markdownguide.org/basic-syntax/#horizontal-rules